### ISSUE TITLE
Unknown Plant Form Creation

### ISSUE THREAD

**Issue by ElectricGr** [MOJN-Veg/mojn-iu-rpackage#10]
General Thoughts on form (should be duplicated for SV)
Need to have ability to take multiple photos. Two different types, one to many of the unknown species and one of the collected pressed specimen.
Have at least three pages (location info, unknown species info, species determination info). 
**Location info** would be PARK, Date, Plot, Transects?, SOP?, Observer, Unk Code
**Unknown species info** would include a Y/N for collected?, best guesses (family, genus, and species), Height if entire plant not collected, Gen info (roots, stem), leaves (present, type, leaflet #, margins), Infl info (Infl type, flowers present?, color, petal #, gen info), Fruit (present?, gen info), Habitat info.
**Species determination info** Final determination (Species Lu), Determined By (contact Lu), Determined Date.
Could add some tracking info to the Species determination form. Unk code updated in database Y/N/NA and date, Data updated on datasheet Y/N/NA (would almost always be NA) and date. Maybe some info about what is being done with the collection (destroyed, accessioned, ect.) While be don't currently have permission to accession specimens, that could change and I'm sure LAKE would take any good specimens from LAKE for their inhouse herbarium.

**Comment by wright13** [comment]
@ElectricGr Does the photo of the pressed specimen happen in the field? Or back in the office when the final determination is being made?

**Comment by ElectricGr** [comment]
Yes, that photo is taken in the field. It's just to have a may to associate the collected plant with the unk code incase it falls out of the newspaper or something else happens.

**Comment by wright13** [comment]
- [ ] Make general photo required if no collection

**Comment by ElectricGr** [comment]
@HarpoMark 

A few additional changes/ updates for the Unk Species Form

- [ ] Add PARA 2023 to field season.
- [ ] Remove observer. This is a 1 person protocol. Change recorder to Collector (single select).
- [ ] Update outbox naming convention. Plot, unk number (add unk to beginning), final determination (yes/no) field season/Year. Example PARAC010_UNK4_No_2023
- [ ] Single select for in-field review
